src/build_workflow/build_recorder.py

```diff
@@ -75,6 +81,7 @@ def append_component(self, name: str, version: str, repository_url: str, ref: st
                 "version": version,
             }
             self.components_hash[name] = component
+            logging.info(f"Appended {name} component in input manifest.")
```

Comment:
Should this be build manifest?

Reply:
Good catch. Updated in the latest commit. Thanks.